app/static-data/actions.json

```diff
@@ -18,7 +18,8 @@
       "amountPerHectare": 129
     },
     "eligibilityRules": [
-      { "id": "is-below-moorland-line" }
+      { "id": "is-below-moorland-line" },
+      { "id": "is-within-parcel-area" }
```

Comment:
I actually don't think this should be a rule in this sense. When would you ever want to not include it? It's more just a reality check in code I think

Reply:
There are cases where it's not needed. For example SAM1 and CSAM1 have a rule in place which enforces that entire parcel area be used rather than an areaAppliedFor <= the parcel area.

Reply:
I could be missing something here but if the entire parcel area is used then areaAppliedFor == parcel area right? When would we ever allow areaAppliedFor > parcel area?

Reply:
yeah, I agree. It feels like 'is within parcel area' should be a default. Surely anything larger than the parcel area would be an exception